source/renderer/app/containers/Root.js

```diff
@@ -59,7 +61,7 @@ export default class Root extends Component<Props> {
       (isPageThatDoesntNeedWallets && !isNodeInStoppingSequence) ||
       (isProfilePage && (isNotEnoughDiskSpace || !isNodeInStoppingSequence))
     ) {
-      return React.Children.only(children);
+      return <>{children}</>;
```

Comment:
@tomothespian In case children is not one element, it throws an error.
That was used to ensure it is returning only one root element.
This was updated due to updated router.
Also `<>` is new way of adding `Fragment`, shorthand version.
For more info: https://reactjs.org/docs/fragments.html

Reply:
@yakovkaravelov I know that `<>` is shorthand for Fragment. Maybe we should handle what we are passing as children on top-level, not here. But that is not connected to this task. Just asked

Reply:
@tomothespian After upgrading react router, above `children` sometimes become array of elements, not single root element which was not the case in previous router.
That's why I removed `React.Children.only` part.
If you test this on local, you will find what it would look like.

Reply:
👍